core/src/main/java/io/fabric8/maven/core/util/SpringBootProperties.java

```diff
@@ -23,5 +23,9 @@
     public static final String SERVER_PORT = "server.port";
     public static final String SERVER_KEYSTORE = "server.ssl.key-store";
     public static final String DEV_TOOLS_REMOTE_SECRET = "spring.devtools.remote.secret";
+    public static final String DEV_TOOLS_REMOTE_SECRET_ENV = "SPRING_DEVTOOLS_REMOTE_SECRET";
     public static final String CONTEXT_PATH = "server.context-path";
+    public static final String SPRING_BOOT_GROUP_ID = "org.springframework.boot";
+    public static final String SPRING_BOOT_ARTIFACT_ID = "spring-boot";
+    public static final String SPRING_BOOT_DEVTOOLS_ARTIFACT_ID = "spring-boot-devtools";
```

Comment:
Yeah, it was one of my proposals few months ago... Not sure if this can be helpful, I think a more lightweight separation (like packages) can be easier to implement.. I've found some problems in an attempt to define things as core services because of cyclic dependencies, maybe we better keep the number of modules low..